dotcom-rendering/src/web/lib/renderContainer.tsx

```diff
@@ -0,0 +1,53 @@
+import { ContainerLayout } from '../components/ContainerLayout';
+import { DynamicFast } from '../components/DynamicFast';
+import { FixedLargeSlowXIV } from '../components/FixedLargeSlowXIV';
+
+type Props = {
+	trails: DCRFrontCard[];
+	containerType: DCRContainerType;
+};
+
+const renderContainer = ({ trails, containerType }: Props) => {
+	switch (containerType) {
+		case 'dynamic/fast':
+			return <DynamicFast trails={trails} />;
+		case 'fixed/large/slow-XIV':
+			return <FixedLargeSlowXIV trails={trails} />;
+		default:
+			// TODO: This default allows us to render fronts in-development where we might
+			// not support all the container types, but it should be removed / re-investigated
+			// before fronts are released
+			return <DynamicFast trails={trails} />;
```

Comment:
I would be tempted to have a `<Debug trails={trails}>` container as the default, at least in development. This could also tie in some tracking or graph, where we can see the remaining unsupported containers.

Reply:
I think this could be really useful, but I wonder if while we don't support many container types, we would potentially end up with most fronts rendering no cards at all which might not be as useful for development

Reply:
In terms of logging container types we don't support, this might be easier done in Frontend?

Reply:
Maybe we can capture this in ELK?

Reply:
Yes, this would be entirely appropriate for ELK, but I’m not sure we’ve got the pipeline set up for this yet. Part of what the data bubble will look into.